tests/test_bs_helper.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+import sys
+import os
+
+from unittest.mock import patch, PropertyMock
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                             '..')))
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                             '..', 'smarsy')))
+# excluding following line for linter as it complains that
+# from import is supposed to be at the top of the file
+
+from smarsy.bs_helper import BSHelper # noqa
+
+
+class TestBSHelperInstance(unittest.TestCase):
+    def setUp(self):
+        self.html = 'some html'
+        self.source_page = BSHelper(self.html)
+
+    def test_bshelper_instance_created(self):
+        self.assertEqual(self.source_page.html, self.html)
+
+
+class TestGetPageSource(unittest.TestCase):
```

Comment:
If you are using test class to separate tests for specific object, then add it to the class name. `Test_get_bs_object`. Naming conventions for the test names and their classes are different than for other objects. The same applies to the all test classes below.

Reply:
Done